### ISSUE TITLE
Print absolute raw bits 1,0s no parsing

### ISSUE THREAD

**Issue by EdwinFairchild** [JiaoXianjun/BTLE#26]
0. Who are you?
    _Application Engineer_
1. What is your own modification (if any)?
    _None_
2. Environments: OS type/version, BTLE repo branch and commit revision
    _Ubuntu Latest. BTLE main branch_
3. Board/hardware type
    _HACKRF_
4. Detailed commands/steps to reproduce the issue, and the related error message, screenshot, etc
_NONE_
5. Your debug efforts and results (if any)
_NA_
6. Any other thing we need to know for helping you better?
_NA_

This is not an issues, but you do not have discussions enabled, so I will ask my question here.
Question 1.
How would I modify your code in order to be able to print a bit stream of BLE traffic , absolute zero parsing ?. Or for more general purpose, how can I print the bits of what ever rf signal  hackrf is giving me. ? Essentialy I am trying to demodulate a signal very similar to BLE and I need to see the raw data to make sure we are transmitting properly. Your work here seems like it can do the job I am just unsure how to do so. 
Question 2.
I noticed the `-r` flag for btle_rx but is that really absolute raw? or are you parsing header and aa because is see it displayed there.?

**Comment by JiaoXianjun** [comment]
You can print the bit decision here: https://github.com/JiaoXianjun/BTLE/blob/5fbbead53c5758a9042f55fed80d7c0780d10ef1/host/btle-tools/src/btle_rx.c#L1382

**Comment by EdwinFairchild** [comment]
I went to `demod_byte` and I am printing 1,0s from there but I am not sure if this is indeed demodulaating the entirety of the signal or if something else is being stripped from the signal

```
void demod_byte(IQ_TYPE* rxp, int num_byte, uint8_t *out_byte) {
  int i, j;
  int I0, Q0, I1, Q1;
  uint8_t bit_decision;
  int sample_idx = 0;
  
  for (i=0; i<num_byte; i++) {
    out_byte[i] = 0;
    for (j=0; j<8; j++) {
      I0 = rxp[sample_idx];
      Q0 = rxp[sample_idx+1];
      I1 = rxp[sample_idx+2];
      Q1 = rxp[sample_idx+3];
      bit_decision = (I0*Q1 - I1*Q0)>0? 1 : 0;
      out_byte[i] = out_byte[i] | (bit_decision<<j);

      sample_idx = sample_idx + SAMPLE_PER_SYMBOL*2;
      printf("%d",bit_decision);
    }
  }
  printf("\r\n");
}
```

**Comment by JiaoXianjun** [comment]
Indeed, that is after the unique word hitting. You need to go to search_unique_bits function to understand how that works. That works for the raw streaming iq and tries to find the unique word in the the incoming raw iq.

**Comment by EdwinFairchild** [comment]
Thanks , I will look into it.

**Comment by EdwinFairchild** [comment]
Ok, @JiaoXianjun  is there anyway I can get this level of data , bits of the entire packet. 
![image](https://user-images.githubusercontent.com/62710807/181355319-1904d484-aa33-4901-9405-7310c0f80832.png)
Obviously this is done with an scope or signal analyzer. 
But could I accomplish this with your code ?

**Comment by JiaoXianjun** [comment]
Yes you can.

**Comment by EdwinFairchild** [comment]
I have accomplished to do this much.
![image](https://user-images.githubusercontent.com/62710807/181380139-375436c1-f6e9-4778-9bc7-252817f13d2e.png)
 I am not sure why I feel maybe there should be more data... ? but i could be wrong

**Comment by EdwinFairchild** [comment]
I am unsure on how to go about this, i just do not understand a lot of it. Specially the IQ things. I noticed some preamble things  in your code but its commented out. and the array is huge so im guessing its not decoded. 
Any help  would be welcomed and appareicated
